vars/RedHat.yml

```diff
@@ -7,3 +7,39 @@ libvirt_vm_log_owner: qemu
 libvirt_vm_volume_creation_env:
   VOLUME_GROUP: qemu
   VOLUME_OWNER: qemu
+
+# Packages that are only necessary if you require EFI support
+libvirt_vm_extra_packages_efi:
+  - edk2.git-ovmf-x64 # Official OVMF package doesn't boot (CentOS 7.5)
+  - qemu-kvm-ev # Need smm support for secure boot
```

Comment:
Does this require hardware virtualisation?

Reply:
I believe SMM emulation, which is needed for secure boot, requires either KVM or TCG acceleration to be enabled. So, by enabling TCG we should be able to use secure-boot without KVM - but I will need to test this to be sure.